src/main/java/services/services_factory/BasicObservableRepositoryFactory.java

```diff
@@ -0,0 +1,36 @@
+package services.services_factory;
+
+import data_gateway.event.ObservableEventEntityManager;
+import data_gateway.event.ObservableEventRepository;
+import data_gateway.task.ObservableTaskEntityManager;
+import data_gateway.task.ObservableTaskRepository;
+
+public class BasicObservableRepositoryFactory implements ObservableRepositoryFactory {
```

Comment:
What are the purposes of the basic factories? If they have specific purposes, maybe consider naming the classes more explicitly

Reply:
The Basic- prefix was just to get around the issue of the interface having the same name. There's not really much else to say other than it's the observable repository factory. Maybe that it delegates to another factory? DecoratingObservableRepositoryFactory idk xD

Reply:
I see